backend/dataall/core/permissions/services/tenant_policy_service.py

```diff
@@ -121,6 +123,26 @@ def validate_permissions(session, tenant_name, g_permissions, group):
         return tenant_group_permissions
 
 
+class TenantActionsService:
+    @staticmethod
+    def update_monitoring_ssm_parameter(name, value):
+        # raises UnauthorizedOperation exception, if there is no admin access
+        context = get_context()
+        TenantPolicyValidationService.validate_admin_access(
+            context.username, context.groups, 'UPDATE_SSM_PARAMETER_MONITORING'
+        )
+
+        current_account = SessionHelper.get_account()
+        region = os.getenv('AWS_REGION', 'eu-west-1')
+        response = ParameterStoreManager.update_parameter(
+            AwsAccountId=current_account,
+            region=region,
+            parameter_name=f'/dataall/{os.getenv("envname", "local")}/quicksightmonitoring/{name}',
+            parameter_value=value,
+        )
```

Comment:
I know this is carry over from resolver code - but this does not work...

All of the following use ParameterStore and try to assume into pivotRole in infra account to get SSM (pivot role DNE in infra account)
  createQuicksightDataSourceSet,
  getMonitoringDashboardId,
  getMonitoringVPCConnectionId,
  updateSSMParameter


Will approve this PR but leave this comment in https://github.com/data-dot-all/dataall/pull/1712/files where I think more focus is on QuickSight Monitoring View